test/import/FSHTank.test.ts

```diff
@@ -151,6 +225,15 @@ describe('FSHTank', () => {
     doc3.invariants.set('Invariant1', new Invariant('Invariant1'));
     doc3.invariants.get('Invariant1').description = 'first invariant';
     doc3.invariants.get('Invariant1').severity = new FshCode('error');
+    // TODO - add back in once rules on invariants are supported
+    // const invariant2 = new Invariant('Invariant2');
+    // invariant2.description = 'second invariant';
+    // invariant2.severity = new FshCode('error');
+    // const invariant2Version = new CaretValueRule('');
+    // invariant2Version.caretPath = 'version';
+    // invariant2Version.value = 'v-2.0.0';
+    // invariant2.rules.push(invariant2Version);
+    // doc3.invariants.set(invariant2.name, invariant2);
```

Comment:
I think you can delete this since invariants don't have a version.

Reply:
Part of [0d0852d](https://github.com/FHIR/sushi/pull/1275/commits/0d0852d19cbf5a24e3e63fdf9a7f359ac288bcce)